test/lib/testHelpers.ts

```diff
@@ -83,6 +83,12 @@ export const advanceToNextEpoch = async (epochManager: EpochManager): Promise<vo
   return advanceBlocks(epochLen.sub(blocksSinceEpoch))
 }
 
+export const advanceEpochs = async (epochManager: EpochManager, n: number): Promise<void> => {
+  for (let i = 0; i < n + 1; i++) {
```

Comment:
Isn't this advancing `n+1` epochs? i.e. if `n == 1`, this will loop twice.

Reply:
oh good catch